packages/manager/modules/pci/src/projects/project/training/jobs/info/translations/Messages_fr_FR.json

```diff
@@ -1,7 +1,9 @@
 {
   "pci_projects_project_training_job_logs_text_1": "Les dernières 100 lignes de logs sont visibles dans cet onglet",
   "pci_projects_project_training_job_logs_text_2": "Pour accéder à l'integralité des logs du job via la ligne de commande, lancer la commande suivante :",
-  "pci_projects_project_training_job_logs_text_3": "Une fois le job terminé, vous pouvez télécharger vos logs via l'object storage, dans le container \"ai-training-logs\"",
+  "pci_projects_project_training_job_logs_text_3": "Le job vient de se terminer. Quand la synchronisation avec l'object storage sera finalisée vous pourrez télécharger vos logs via le container \"ai-training-logs\" de la région GRA.",
```

Comment:
Should the region be included in the translation? Cannot change at some point?
Keep in mind that it could be complicated for the Translation Team to make sure that they must not change its name.

Reply:
Region removed from the translation, espacially since it depends on where the job was run